runtime/moonbase/src/lib.rs

```diff
@@ -816,6 +844,355 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+parameter_types! {
+	// The network Id of the relay
+	pub const RelayNetwork: NetworkId = NetworkId::Polkadot;
```

Comment:
For moonbase the relay network is not Polkadot ?

Reply:
related to my previous answer